packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -396,6 +425,32 @@ class _DraggableState<T> extends State<Draggable<T>> {
   }
 }
 
+/// Represents the details when a specific pointer event occurred on
+/// the [Draggable].
+///
+/// This includes the [Velocity] at which the pointer was moving and [Offset]
+/// when the draggable event occurred, and whether its [DragTarget] accepted it.
+///
+/// Also, this is the Details object for callbacks that use [DragEndCallback].
+class DraggableDetails {
+  DraggableDetails._({
```

Comment:
constructor will need documentation when public

Reply:
Added documentation to the constructor now that it's been made public.